styles/Components.module.css

```diff
@@ -0,0 +1,148 @@
+.compContainer {
+  background: #f6f6fa;
+  padding-left: 85px;
+  padding-right: 69px;
+  display: flex;
+  justify-content: space-between;
+  padding-top: 74px;
+}
+
+.compTitle {
+  font-family: Inter;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 32px;
```

Comment:
These are the changes your going to hate me for lol. Can we move all of these font's and colors etc. into global.css? Exactly how words alive does it, so we can just do `composes from` in the CSS file. It will make everyone's life much easier.